src/plugins/kibana_usage_collection/server/collectors/management/schema.ts

```diff
@@ -11,120 +11,405 @@ import { UsageStats } from './types';
+  'timelion:quandl.key': {
+    type: 'keyword',
+    _meta: { description: 'timelion:quandl.key was changed.' },
+  },
+  'securitySolution:defaultIndex': {
+    type: 'keyword',
+    _meta: { description: 'securitySolution:defaultIndex was changed.' },
+  },
+  'securitySolution:newsFeedUrl': {
+    type: 'keyword',
+    _meta: { description: 'securitySolution:newsFeedUrl was changed.' },
+  },
+  'xpackReporting:customPdfLogo': {
+    type: 'keyword',
+    _meta: { description: 'xpackReporting:customPdfLogo was changed.' },
+  },
+  'notifications:banner': {
+    type: 'keyword',
+    _meta: { description: 'notifications:banner was changed.' },
+  },
+  'timelion:graphite.url': {
+    type: 'keyword',
+    _meta: { description: 'timelion:graphite was changed.' },
+  },
+  'xpackDashboardMode:roles': {
+    type: 'keyword',
+    _meta: { description: 'xpackDashboardMode:roles was changed.' },
+  },
+  'securitySolution:ipReputationLinks': {
+    type: 'keyword',
+    _meta: { description: 'securitySolution:ipReputationLinks was changed.' },
+  },
   // non-sensitive
-  'visualize:enableLabs': { type: 'boolean' },
-  'visualization:heatmap:maxBuckets': { type: 'long' },
-  'visualization:colorMapping': { type: 'text' },
-  'visualization:regionmap:showWarnings': { type: 'boolean' },
-  'visualization:dimmingOpacity': { type: 'float' },
-  'visualization:tileMap:maxPrecision': { type: 'long' },
-  'csv:separator': { type: 'keyword' },
-  'visualization:tileMap:WMSdefaults': { type: 'text' },
-  'timelion:target_buckets': { type: 'long' },
-  'timelion:max_buckets': { type: 'long' },
-  'timelion:es.timefield': { type: 'keyword' },
-  'timelion:min_interval': { type: 'keyword' },
-  'timelion:default_rows': { type: 'long' },
-  'timelion:default_columns': { type: 'long' },
-  'timelion:es.default_index': { type: 'keyword' },
-  'timelion:showTutorial': { type: 'boolean' },
-  'securitySolution:timeDefaults': { type: 'keyword' },
-  'securitySolution:defaultAnomalyScore': { type: 'long' },
-  'securitySolution:refreshIntervalDefaults': { type: 'keyword' },
-  'securitySolution:enableNewsFeed': { type: 'boolean' },
-  'search:includeFrozen': { type: 'boolean' },
-  'courier:maxConcurrentShardRequests': { type: 'long' },
-  'courier:batchSearches': { type: 'boolean' },
-  'courier:setRequestPreference': { type: 'keyword' },
-  'courier:customRequestPreference': { type: 'keyword' },
-  'courier:ignoreFilterIfFieldNotInIndex': { type: 'boolean' },
-  'rollups:enableIndexPatterns': { type: 'boolean' },
-  'notifications:lifetime:warning': { type: 'long' },
-  'notifications:lifetime:banner': { type: 'long' },
-  'notifications:lifetime:info': { type: 'long' },
-  'notifications:lifetime:error': { type: 'long' },
-  'doc_table:highlight': { type: 'boolean' },
-  'discover:searchOnPageLoad': { type: 'boolean' },
+  'visualize:enableLabs': {
+    type: 'boolean',
+    _meta: { description: 'Non-default value of visualize:enableLabs' },
```

Comment:
Super-NIT: Bearing in mind that analysts don't necessarily need to know the Kibana internals, I wonder if it'd make sense to explain what the setting is about. I don't think these changes should block this PR though :)

Reply:
We can improve on these over time.